pkg/sql/exec_util.go

```diff
@@ -190,6 +190,15 @@ var traceSessionEventLogEnabled = settings.RegisterBoolSetting(
 	false,
 ).WithPublic()
 
+// debugForceRealSpans can be used to force the use of real tracing spans for
+// every statement. It's used primarily to measure the tracing overhead in
+// BenchmarkTracing.
+var debugForceRealSpans = settings.RegisterBoolSetting(
```

Comment:
May I suggest a debug knob instead of a cluster setting. The cluster setting increases the chance that users will shoot themselves in the foot.